src/layouts/default.vue

```diff
@@ -110,7 +110,7 @@ export default embeddedPage
   .main {
     height: 100%;
     display: grid;
-    grid-template-columns: 1fr 316px;
+    grid-template-columns: 1fr 336px;
```

Comment:
What is the reason for this change? The sidebar is 316 px in the mockups.

Reply:
Adding CC to the license names caused the names to wrap in the sidebar. Adding a few pixels allow the names to fit in one line each.